pkgs/tools/backup/autorestic/default.nix

```diff
@@ -1,17 +1,17 @@
-{ lib, fetchFromGitHub, installShellFiles, buildGoModule }:
+{ lib, fetchFromGitHub, installShellFiles, buildGo118Module }:
```

Comment:
rather than swapping buildGoModule here, you could do it in all-packages similar to https://github.com/NixOS/nixpkgs/pull/165041/files#diff-ab5748dc9567516fefba8344056b51ec1866adeace380f46e58a7af3d619ea22